Description
===========
Add user_bh so the spacing around the bar font can be configured.

From dwm-bar-height-spacing-6.3.diff. Context refreshed only.

Applies to
==========
dwm git master, commit 44dbc68 (2026-03-13)

diff a/config.def.h b/config.def.h
--- a/config.def.h	2026-03-13 18:27:18.000000000 +0100
+++ b/config.def.h	2026-08-23 13:09:19.985495694 +0200
@@ -5,6 +5,7 @@
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
+static const int user_bh            = 2;        /* 2 is the default spacing around the bar's font */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
 static const char col_gray1[]       = "#222222";
diff a/dwm.c b/dwm.c
--- a/dwm.c	2026-03-13 18:27:18.000000000 +0100
+++ b/dwm.c	2026-08-23 13:09:19.985591352 +0200
@@ -1562,7 +1562,7 @@
 	if (!drw_fontset_create(drw, fonts, LENGTH(fonts)))
 		die("no fonts could be loaded.");
 	lrpad = drw->fonts->h;
-	bh = drw->fonts->h + 2;
+	bh = drw->fonts->h + user_bh;
 	updategeom();
 	/* init atoms */
 	utf8string = XInternAtom(dpy, "UTF8_STRING", False);
